sklearn/datasets/twenty_newsgroups.py

```diff
@@ -216,11 +216,17 @@ def fetch_20newsgroups(data_home=None, subset='train', categories=None,
 
     Returns
     -------
-    bunch : Bunch object
-        bunch.data: list, length [n_samples]
-        bunch.target: array, shape [n_samples]
-        bunch.filenames: list, length [n_classes]
-        bunch.DESCR: a description of the dataset.
+    bunch : Bunch object with the following attribute:
+        - bunch.data: list, length [n_samples]
+
```

Comment:
Do we need all these blank lines?

Reply:
Ready. I remove all blank lines from the docs. I did not sure about the sphinx behavior, because of that I used the blank line.  :-)